vmm/src/device_manager.rs

```diff
@@ -2107,6 +2113,12 @@ impl DeviceManager {
         self.hotplug_virtio_pci_device(device, iommu_attached)
     }
 
+    #[cfg(feature = "pci_support")]
+    pub fn add_fs(&mut self, fs_cfg: &mut FsConfig) -> DeviceManagerResult<()> {
+        let (device, iommu_attached) = self.make_virtio_fs_device(fs_cfg)?;
+        self.hotplug_virtio_pci_device(device, iommu_attached)
+    }
```

Comment:
This doesn't build add the moment. The signature of `hotplug_virtio_pci_device` changed in the previous pr. Mapping the result value to `()` should resolve the issue.